packages/http/src/router.ts

```diff
@@ -413,12 +401,26 @@ export class Router {
 
     protected routes: RouteConfig[] = [];
 
-    //todo, move some settings to KernelConfig
-    protected form = formidable({
-        multiples: true,
-        hash: 'sha1',
-        enabledPlugins: ['octetstream', 'querystring', 'json'],
-    });
+    private parseBody(req: HttpRequest, files: { [name: string]: UploadedFile }) {
+        const form = formidable({
+            multiples: true,
+            hash: 'sha1',
+            enabledPlugins: ['octetstream', 'querystring', 'json'],
+        });
+        return asyncOperation((resolve, reject) => {
+            if (req.body) {
+                return resolve(req.body);
+            }
+            form.parse(req, (err: any, fields: any, files: any) => {
+                if (err) {
+                    reject(err);
+                } else {
+                    const body = req.body = { ...fields, ...files };
```

Comment:
This makes it possible to:
a) re-use other outside of deepkit uses of node-formidable (for advanced users)
b) or potentially run parseBody twice